### PR TITLE
SEQNG-314: New seqexec release

### PR DESCRIPTION
**Pull request by cquiroz**
configuration for seqexec GN production and version update

### CODE REVIEW

File: app/seqexec-server-gn/src/main/resources/app.conf [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#
+# Seqexec server configuration for production at GS
```
Comment (jluhrs) [inline app/seqexec-server-gn/src/main/resources/app.conf]: GS?

File: app/seqexec-server-gn/src/main/resources/app.conf [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    gcalSim = false
+    odbNotifications = true
+    tcsKeywords = true
+    f2Keywords = false
+    gwsKeywords = true
+    gcalKeywords = true
+    odbQueuePollingInterval = "3 seconds"
+    tops = "tcs=tcs:, ao=ao:, gm=gm:, gc=gc:, gws=ws:, m2=m2:, oiwfs=oiwfs:, ag=ag:, f2=f2:"
+    epics_ca_addr_list = "10.2.2.255 10.2.10.21 10.2.126.101"
+    smartGCalHost = "gsodb.gemini.edu"
```
Comment (jluhrs) [inline app/seqexec-server-gn/src/main/resources/app.conf]: gsodb?

Comment (cquiroz) [inline app/seqexec-server-gn/src/main/resources/app.conf]: That is actually correct, smartGCal must refer to gsodb

File: app/seqexec-server-gn/src/main/resources/app.conf (hunk 1 above)
Comment (cquiroz) [inline app/seqexec-server-gn/src/main/resources/app.conf]: Fixed

**Pull request merged**